doc/codebase-structure-and-guidelines.md

```diff
@@ -78,3 +87,52 @@ These guidelines make db.cljs namespaces the place to go when making changes to
   - `:module.ui/` for user triggered events
   - `:module.callback/` for callback events, which are events bringing back the result of an fx to the event loop, the name of the event should end with `-success` or `-error` most of the time. Other possibilities can be `-granted`, `-denied` for instance.
   - `:module/` for internal events, examples are time based events marked `-timed-out`, external changes marked `-changed` or reception of external events marked `-received`.
+
+
+## Testing flow
+- All PRs automatically go to "REVIEW" column on [Pipeline for QA](https://github.com/status-im/status-react/projects/7) project on Github. This is our main board for QA / devs interaction
+- After the PR gets at least 1 approval, it should be moved to "E2E Tests" column. Some PRs may need approvals from more than one person.
+- Critical path tests are automatically run for all PRs in "E2E Tests" column.
+- If E2E tests pass:
+  - If PR doesn't need manual QA (it is small change or change doesn't relate to app) - after passing e2e tests it can be merged (don't use the Github UI to merge, check section on merging PRs).
+
+  - If manual QA needed - add label 'request manual qa'. The QA team will process it (make sure the summary and test instructions are clear). QA team will review results and test PR on different platforms and assign one of the three labels:
+    - No issues - label 'TESTED-OK' and then you can merge it.
+    - If issues - label 'TESTED-ISSUES' and comment from QA with bugs.
+    - After fix and/or discussion the process is repeated.
+
+  - If manual QA is not needed but all tests don't pass, you can ping @churik or @Serhy to confirm that failed E2E tests are not unrelated. 
+
+## Enabling debug logs
+Calls to `log/debug` will not be printed to the console by default. It can be enabled under "Advanced settings" in the app:
+
+![Enable Debug Logs](https://media.discordapp.net/attachments/626738033701683200/778912893105995786/unknown.png)
```

Comment:
not sure this link will be always available

Reply:
Uploaded image to this repo and created a local link.